packages/expo-location/build/Location.types.d.ts

```diff
@@ -194,6 +194,10 @@ export type LocationTaskServiceOptions = {
      * Boolean value whether to destroy the foreground service if the app is killed.
      */
     killServiceOnDestroy?: boolean;
+      /**
+   * Boolean value whether to show the notification in the status bar.
+   */
```

Comment:
```suggestion
    /**
     * Boolean value whether to show the notification in the status bar. Defaults to false.
     *
     * @android
     */
```
It looks like this change ended up in the wrong spot! Instead of editing files in the `build` folder, it would be great if you could move this change to the correct file in the `src` directory. After that, just rebuild the JavaScript files by running `yarn build` inside the `expo-location` folder.